crates/control-interface/src/lib.rs

```diff
@@ -850,7 +826,7 @@ mod tests {
             .timeout(Duration::from_millis(1000))
             .auction_timeout(Duration::from_millis(1000))
             .build();
-        let mut receiver = client.events_receiver().await.unwrap();
+        let mut receiver = client.events_receiver("foobar".to_string()).await.unwrap();
```

Comment:
this just needs to be `vec!["foobar".to_string()]`

Reply:
Whoops! TY